subjects/ascii-art/justify/README.md

```diff
@@ -13,8 +13,7 @@ You must follow the same [instructions](https://public.01-edu.org/subjects/ascii
 
 This project will help you learn about :
 
-- Client utilities.
-- The Go file system(**fs**) API.
+- The Go file system(**fs**).
```

Comment:
I think it was better with API, or maybe library or functions ?
Why did you decided to remove it ?
It's not like it's the go filesystem, a language do not provide a file system.

Reply:
i see an API more like a "software" that has no connection to a language. Maybe library or package is more suitable, what do you think?

Reply:
API is how you interact with it, code is an API for reference `https://nodejs.org/api/fs.html` this is the link of the node API which contains code to interact with the FS, you can look at occurrences of `API` in the page and find quite a lot.

Reply:
ooooh ok, got it, then i will leave API in there